src/main/cpp/pop/Model.h

```diff
@@ -0,0 +1,113 @@
+#ifndef POPULATION_MODEL_H_INCLUDED
+#define POPULATION_MODEL_H_INCLUDED
+
+/**
+ * @file
+ * Header file for the population model class
+ */
+
+#include <memory>
+#include <vector>
+#include <boost/property_tree/ptree.hpp>
+#include "util/InclusiveRange.h"
+
+namespace stride {
+namespace population {
+
+struct TownPivot final
```

Comment:
Done! I renamed it to a `TownRange`, which… still sounds like a range of towns, or something. It’s kind of a nebulous thing that doesn’t make sense to give a name to :( `Pivot` was just a bad translation (based on “spilcijfers”)  on my end, though.